### PR TITLE
fix bug search

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/screen/searchpage/SearchContainerViewModel.java [hunk 1]
```diff
@@ -87,4 +86,9 @@ public void onSearchAction(String keyWord) {
             }
         };
     }
+
+    @Bindable
+    public boolean getClear() {
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchpage/SearchContainerViewModel.java]: WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName

File: app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java [hunk 2]
```diff
@@ -57,11 +63,16 @@ public void onAddToCartSuccess() {
...
 
     @Override
-    public void onAddToCartError(Throwable e) {
+    public void onError(Throwable e) {
         mNavigator.showToast(e.getMessage());
     }
 
     @Override
+    public void onOrderProductSuccess() {
+        mNavigator.showToastCustomActivity(R.string.order_successful);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java]: ERROR: [PMD-CPD] 25 lines duplicated.
app/src/main/java/com/framgia/forder/screen/listProduct/ListProductViewModel.java

File: app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java [hunk 3]
```diff
@@ -83,4 +89,35 @@ public void onItemRecyclerViewClick(Object item) {
                     "ProductDetailFragment");
         }
     }
+
+    @Override
+    public void onAddToCartSuccess() {
+        mNavigator.showToast(R.string.add_to_cart_success);
+    }
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java]: ERROR: [PMD-CPD] 28 lines duplicated.
app/src/main/java/com/framgia/forder/screen/listProduct/ListProductViewModel.java

**Review by tuanlvt-1117 — APPROVED** [review]

File: app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java [hunk 4]
```diff
@@ -83,4 +89,35 @@ public void onItemRecyclerViewClick(Object item) {
                     "ProductDetailFragment");
         }
     }
+
+    @Override
+    public void onAddToCartSuccess() {
+        mNavigator.showToast(R.string.add_to_cart_success);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java]: ERROR: [PMD-CPD] 24 lines duplicated.
app/src/main/java/com/framgia/forder/screen/productdetail/ProductDetailViewModel.java

File: app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java [hunk 5]
```diff
@@ -83,4 +95,35 @@ public void onItemRecyclerViewClick(Object item) {
                     "ProductDetailFragment");
         }
     }
+
+    @Override
+    public void onAddToCartSuccess() {
+        mNavigator.showToastCustomActivity(R.string.add_to_cart_success);
+    }
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java:102]: ERROR: [PMD-CPD] 28 lines duplicated.
app/src/main/java/com/framgia/forder/screen/listProduct/ListProductViewModel.java

File: app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java [hunk 6]
```diff
@@ -83,4 +95,35 @@ public void onItemRecyclerViewClick(Object item) {
                     "ProductDetailFragment");
         }
     }
+
+    @Override
+    public void onAddToCartSuccess() {
+        mNavigator.showToastCustomActivity(R.string.add_to_cart_success);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultViewModel.java:101]: ERROR: [PMD-CPD] 24 lines duplicated.
app/src/main/java/com/framgia/forder/screen/productdetail/ProductDetailViewModel.java

**Pull request merged**